src/components/ProductTours/ProductTours.jsx

```diff
@@ -29,15 +29,23 @@ const ProductTours = ({
   enableLearnerPortal,
 }) => {
   const { FEATURE_CONTENT_HIGHLIGHTS } = getConfig();
+  const [tours, setTours] = useState([]);
   const enablePortalAppearance = features.SETTINGS_PAGE_APPEARANCE_TAB;
   const history = useHistory();
-  const enabledFeatures = {
-    [PORTAL_APPEARANCE_TOUR_COOKIE_NAME]: usePortalAppearanceTour({ enablePortalAppearance })[0],
-    [BROWSE_AND_REQUEST_TOUR_COOKIE_NAME]: useBrowseAndRequestTour({ enableLearnerPortal })[0],
-    [LEARNER_CREDIT_COOKIE_NAME]: useLearnerCreditTour()[0],
-    [HIGHLIGHTS_COOKIE_NAME]: useHighlightsTour(FEATURE_CONTENT_HIGHLIGHTS)[0],
-  };
-  const newFeatureTourCheckpoints = {
+
+  const portalAppearance = usePortalAppearanceTour({ enablePortalAppearance });
+  const browseAndRequest = useBrowseAndRequestTour({ enableLearnerPortal });
+  const learnerCredit = useLearnerCreditTour();
+  const highlightTour = useHighlightsTour(FEATURE_CONTENT_HIGHLIGHTS);
+
+  const enabledFeatures = useMemo(() => ({
```

Comment:
[curious/clarification] Are the changes in this component (beyond changing the return value of the `usePortalAppearanceTour`, `useBrowseAndRequestTour`, etc. hooks) still needed now that these hooks no longer make use of state?

I believe keeping this file as it exists on `master` alongside the changes to the hooks (removing state) still seems to resolve the issue (without requiring a re-render when calling `setTours`).

Reply:
Updated the PR and tested multiple routes of it possibly re-rendering the product tours component. Identical behavior to the first fix in the PR with a cleaner outcome 👍🏽